dockstore-webservice/src/main/java/io/swagger/api/impl/ApiVersionConverter.java

```diff
@@ -156,23 +148,18 @@ private static Response getResponse(Object object, MultivaluedMap<String, Object
     }
 
     private static MetadataV1 getMetadataV1(Metadata metadata) {
-        MetadataV1 metadataV1 = new MetadataV1();
-        metadataV1.setMetadata(metadata);
-        return metadataV1;
+        return new MetadataV1(metadata);
     }
 
     private static MetadataV2 getMetadataV2(Metadata metadata) {
-        MetadataV2 metadataV2 = new MetadataV2();
-        metadataV2.setMetadata(metadata);
-        return metadataV2;
+        return new MetadataV2(metadata);
     }
 
+    private static void handleError() {
+        throw new RuntimeException("Unknown");
```

Comment:
Seems pretty useless. Get rid of this method and throw more appropriate CustomWebApplicationExceptions where this is called